packages/ui/src/theme/css/component/input.scss

```diff
@@ -1,3 +1,5 @@
+@import 'fieldControl';
```

Comment:
Can you make sure we need this import? `@import` makes all variables, mixins, and functions globally accessible. I believe we have done that in `field.scss`. And because of its global nature, `@import` is discouraged in favor of `@use` now. I think we had better looking for a solution using `@use`